_gsocorgs/2018/uoslo.md

```diff
@@ -1,8 +1,11 @@
 ---
 title: "University of Oslo"
-author: "Mario Lassnig"
+author: "Cédric Serfon"
 layout: default
-organization: UOslo
```

Comment:
Please note that if you change the identifier of the university from UOslo to UiO, your proposal:
https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocproposals/2018/proposal_ATLASrucioMetadata.md
will need to point to that new identifier.

Reply:
I change to the new identifier